src/views/includes/subscribe-view.jade

```diff
@@ -36,4 +36,4 @@
     .subscribe__thank-you-wrapper
       h4.headline.headline--m= subscribeView['thanks-title']
       p.subscribe__thank-you-msg!= subscribeView['thanks-msg'][0]
-      p.subscribe__thank-you-msg!= subscribeView['thanks-msg'][1]
+      //p.subscribe__thank-you-msg!= subscribeView['thanks-msg'][1]
```

Comment:
Use delete instead of commenting out, if you think there is a little chance we will need it in future. There is a history to find it and get it back if needed.